app/product/[handle]/page.tsx

```diff
@@ -76,34 +74,25 @@ export default async function ProductPage({ params }: { params: { handle: string
   };
 
   return (
-    <div>
+    <div className="px-4">
```

Comment:
Can this be changed to a fragment and the style moved to the `div` below the script?